import cn.hutool.core.exceptions.ExceptionUtil;
public class FunctionUse1277 {
public void funcUse() {
ExceptionUtil exceptionutil = new ExceptionUtil();
exceptionutil.getCausedBy(null,null);
}
}